StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Embedded/EmbeddedPaymentMethodsView.swift

```diff
@@ -24,12 +24,20 @@ class EmbeddedPaymentMethodsView: UIView {
 
     private let appearance: PaymentSheet.Appearance
     private let rowButtonAppearance: PaymentSheet.Appearance
+    private var previousSelection: Selection?
+    private var previousSelectedRowButton: RowButton?
+    private var selectedRowButton: RowButton? {
+        willSet {
+            previousSelectedRowButton = selectedRowButton
+        }
+    }
     private(set) var selection: Selection? {
+        willSet {
+            previousSelection = selection
+        }
         didSet {
             updateMandate()
-            if oldValue != selection {
-                delegate?.selectionDidUpdate()
-            }
+            delegate?.selectionWasTapped(didUpdate: oldValue != selection)
```

Comment:
I think there are times where the selectionDidUpdate, but a user didn't actually tap (e.g. removal use case), so I think the name selectionDidUpdate() is more appropriate.

Reply:
Updated!